### DISCUSSION TITLE
MacOS

### DISCUSSION THREAD

**Discussion by Durthander** [fexh10/aw-cli#6] [General]
Sarà mai disponibile per Mac os. Ho visto  e provato che con ani-cli funziona molto bene, utilizzando come player IINA, e mi chiedevo se riusciste ad implementalo? Scusate il disturbo

**Comment by fexh10** [comment]
Al momento la compatibilità con MacOS è stata fatta e si trova nel branch develop (potrebbe avere bug). Puoi installarla con ```pip install git+https://github.com/fexh10/aw-cli.git@develop```. Appena tutte le feature in programma saranno concluse, verrà fatta una versione stabile disponibile nel main e direttamente su pip.

**Comment by axtrat** [comment]
Giusto per specificare: i possibili bug non sono relativi alla compatibilità con MacOS ma alle utlime aggiunte.
Inoltre, non è ancora specificato nel README ma, al primo avvio verrà chiesto il path di MPV/VLC che, nel caso sia già presente nella variabile Path d'ambiente (ad esempio se è stato installato tramite ```brew install ...```) ed è quindi possibile avviare il player da teminale, può essere anche omesso e lasciato vuoto, altrimenti va specificato.
